pkg/ignore/ignore_test.go

```diff
@@ -16,15 +16,9 @@ func TestIgnore_Match(t *testing.T) {
 	i := NewIgnore([]string{"my/files/*"})
 	assert.NotNil(t, i)
 
-	assert.False(t, i.Match("not/foo"))
-	assert.True(t, i.Match("my/files/file1"))
-	assert.False(t, i.Match("my/files"))
```

Comment:
I think these should still be included to test that `/` would still match Windows file paths

Reply:
ahh ok i misunderstood the intention of these tests! In that case, would it be better to just abandon this PR?

Reply:
No need to abandon, I think your change adds clarity. Maybe a comment explaining the value of the tests would be helpful?